frog/imports/client/Wiki/index.js

```diff
@@ -255,7 +266,7 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
           (instanceId && instanceId !== this.getInstanceId(fullPageObj)
             ? '/' + instanceName
             : '');
-        this.props.history.push(link);
+        if (!this.props.embedPage) this.props.history.push(link);
```

Comment:
NIT: Add curly braces to single if statement to help with readability

Reply:
Made the respective changes